libc/src/sys/epoll/linux/epoll_create.cpp

```diff
@@ -0,0 +1,38 @@
+//===---------- Linux implementation of the epoll_create function ---------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/sys/epoll/epoll_create.h"
+
+#include "src/__support/OSUtil/syscall.h" // For internal syscall function.
+#include "src/__support/common.h"
+#include "src/errno/libc_errno.h"
+#include <sys/syscall.h> // For syscall numbers.
```

Comment:
What is libc/config/linux/syscall_numbers.h.inc (which also looks like it defines SYS_epoll_create and SYS_epoll_create1)?  I wonder if we should be using that, too, for the same reasons for not using `<sys/epoll.h>` listed above?  Looks like that generates a sys/syscall.h, but I'm guessing that't only for fullbuild mode and not overlay mode?

Reply:
It's marked as a generated header right now, but it doesn't appear to actually need to be. We should look into moving it to /include/ at some point.